packages/web/src/components/Sidebar.tsx

```
@@ -20,64 +31,68 @@ export default function Sidebar() {
       >
-        {open ? "✕" : "☰"}
+        {open ? <X size={24} /> : <Menu size={24} />}
       </button>
 
       {/* Overlay */}
       {open && (
         <div
-          className="fixed inset-0 z-30 bg-black/40 md:hidden"
+          className="fixed inset-0 z-30 bg-black/40 backdrop-blur-sm md:hidden transition-opacity"
           onClick={() => setOpen(false)}
         />
```

Biome (2.3.14)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)